docker-compose.yml

```diff
@@ -66,6 +66,35 @@ services:
     depends_on:
       database:
         condition: service_healthy
+
+  ##########################################################
+  #                      FrontEnd                          #
+  ##########################################################
+  frontend:
+    container_name: quackStack-frontend
+    build:
+      context: ./frontend
+      dockerfile: Dockerfile.dev
```

Comment:
Use a single Dockerfile for Docker Compose, PRs, TEST and PROD.  It's how we ensure image consistency and promotion of the SAME image through all stages.

Reply:
Did you want a hand with this?

Reply:
Yes please

Reply:
Okay!  I'll add a few commits.  Please pull shortly after.  :)